sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AzureMonitorExporterProviderKeys.java

```diff
@@ -0,0 +1,8 @@
+package com.azure.monitor.opentelemetry.exporter.implementation;
+
+public class AzureMonitorExporterProviderKeys {
+
+    public static final String EXPORTER_NAME = "azuremonitor";
```

Comment:
is this consistent with other languages? do we have a spec somewhere?

Reply:
https://teams.microsoft.com/l/message/19:meeting_MGUxMzdjYjctZDJjYi00MmFhLWE5MWUtZWU3NGY2ODFkOWM2@thread.v2/1691164823462?context=%7B%22contextType%22%3A%22chat%22%7D

```suggestion
    public static final String EXPORTER_NAME = "azure_monitor";
```